src/dotnet/commands/dotnet-help/LocalizableStrings.cs

```diff
@@ -79,7 +79,26 @@ internal class LocalizableStrings
 
         public const string CommandArgumentDescription = "CLI command for which to view more detailed help.";
 
+        public const string PathToApplicationDefinition = "The path to an application .dll file to execute.";
```

Comment:
The pattern is like PascalCase the "key" + defination. So the "key" is 
```
path-to-application:
```
So I made it this way